### PR TITLE
Added folder structure and linter set up

### PR DESCRIPTION
**Pull request by Nasirkhan294**
Here are the files & folders added to this pull request:
1.  HTML file
2. CSS file
3. .github/workflow/linters.yml
4. .gitignore file
5. .hintrc file
6. stylelintrc.json file

### CODE REVIEW

**Review by joel-kalema — COMMENTED** [review]
Hi @Nasirkhan294 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that doesn't have a professional README


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      <div class="container">
+        <!-- Start logo section -->
+        <div class="logo-wrapper">
+          <a href="#" class="logo">My Logo</a>
+        </div>
+        <!-- End logo section -->
+        <!-- Start menu section -->
+        <div class="menu-bars" id="menu-toggle">
+          <i class="fa-solid fa-regular fa-bars"></i>
+        </div>
```
Comment (algerina) [inline index.html]: - [ ]  For semantics reasonsthe hamburger button should have a nav tag.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+html {
+  font-size: 16px;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1em;
+}
+
```
Comment (algerina) [inline css/styles.css]: - [ ] The background color should be white as per the Figma design
your preject:
port1

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @Nasirkhan294 ,

Great job so far! 
However you still have a minor issues to fix.

May I highlight the following: 
- [x] Your project looks good👏  ✔️ 
- [x] No Linter errors✔️ 


## STATUS: CHANGES REQUIRED ♻️ 
PLEASE: check the comments under the review



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: index.html (hunk 1 above)
Comment (Nasirkhan294) [reply index.html]: Hi @algerina, I didn't understand what you're emphasizing here. Please explain to me more so I don't lose another review chance. Just let me know where exactly I need to put my nav tag, either inside the "menu-bars" div or inside the (i) tag. Truely appreciate your time and help. Moreover, at this point we are not supposed to add any javascript code so I guess only nav tag is require here if I'm not wrong.

**Review by pinkmoon25 — APPROVED** [review]
Hi @Nasirkhan294 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Readme :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**